src/components/organisms/navigation/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react'
+
+import Header from '_molecules/header'
+import Sidenav from '_organisms/sidenav'
+
+class Navigation extends Component {
+  state = {
+    sidenavIsOpen: false,
+  }
+
+  sideMenuHandler = () => {
+    this.setState(prevState => ({
+      sidenavIsOpen: !prevState.sidenavIsOpen,
+    }))
+  }
+
+  render() {
+    const { sidenavIsOpen } = this.state
+    return (
+      <div>
```

Comment:
why do you need this `<div>`?

Reply:
it is  a `nav`, right?

Reply:
It's just a component that put together the `header` and the `sidenav`